scripts/pin_the_slow_way.py

```diff
@@ -36,6 +36,7 @@
           'icu': 'icu 56.*',
           'jpeg': 'jpeg 9*',
           'libblitz': 'libblitz 0.10|0.10.*',
+          'libevent': 'libevent 2.0.*',
```

Comment:
It is a fair point. However, `libevent` is only used in one dependency that is already pinning it this way.

I would love to see some more discussion on your PR, but expect it will happen after switching to `conda-build` version 2.x.